src/cbc_sdk/workload/vm_workloads_search.py

```diff
@@ -82,6 +83,40 @@ def _build_api_request_uri(self, http_method="GET"):
         """
         return self.urlobject_single.format(self._cb.credentials.org_key, self._model_unique_id)
 
+    def install_sensor(self, sensor_kit_types, config_file=None):
```

Comment:
Instead of a full sensor_kit_type can we just require the sensor version since the other properties are present in the compute resource to make the sensor_kit_type

Reply:
We discussed this but did not get a full resolution.

The logical place to look for this is in the `os_type` and `os_architecture` fields of the ComputeResource.  Which would be all well and good, except for the following:
* These fields are defined as _arrays._ How can or should I "smash them down" to the single value that SensorKit requires?
* There's not a 1:1 correspondence between the potential values of `os_type` and the allowed values for either `device_type` or `type` in SensorKit.  In particular, `os_type` appears to be _missing_ the "MAC" value as a valid value, which would seem to me to be a very serious omission.

Any advice on how to resolve these concerns would be appreciated.

Reply:
I think that is a mistake in the schema portion of the documentation for compute resources as that appears to be defining the properties from the search criteria which supports more than one type.

See the response of the search or get
```{
    "id": "14101309",
    "appliance_uuid": "e66fdd46-2268-4f34-8a3f-226fa6d3edd0",
    "cluster_name": "cwp-bucket-1-cluster",
    "datacenter_name": "cwp-bucket-1-datacenter",
    "esx_host_name": "10.234.56.789",
    "esx_host_uuid": "a2311b42-3e53-8f21-97d7-66680007185f",
    "vcenter_name": "VMware vCenter Server 6.7.0 build-14368073",
    "vcenter_host_url": "10.234.56.789",
    "vcenter_uuid": "9a8a0be5-ae1e-49ce-b2aa-34bc7dc445e3",
    "name": "cwp-bucket-1-appliance",
    "host_name": "photon-machine",
    "created_at": "2020-10-29T16:37:17.610Z",
    "ip_address": "10.234.56.789",
    "eligibility": "UNSUPPORTED",
    "eligibility_code": [
        "Unsupported OS"
    ],
    "installation_status": "NOT_INSTALLED",
    "installation_status_code": null,
    "uuid": "500edcd7-e8ff-868a-6513-4e2fabcfcccd",
    "os_description": "VMware Photon OS (64-bit)",
    "os_type": "OTHER",
    "os_architecture": "32",
    "vmwaretools_version": "11296"
}
```

We should probably also verify the eligibility of the ComputeResource and throw an error if it is not supported.

MAC is likely missing because it is not a supported OS for vcenter so that should not be an issue

Reply:
OK, I have dealt with this as well as the simple-pairs case mentioned in comments below.